### ISSUE TITLE
Case Study revision: barbera

### ISSUE THREAD

**Issue by danielturek** [BIDS/repro-case-studies#45]
Your case study represents a useful and interesting example of reproducibility in applied scientific research, and we value this contribution relating political science, social media, data sharing and privacy.  Your Narrative and Diagram are both excellent, and we're only requesting minor revisions.

Narrative
- [x] Please provide a one sentence “title” for your case study (e.g., “Developing, testing, and deploying efficient MCMC algorithms for hierarchical models using R”, or an equivalent level of detail).  This can be added at the very beginning of your Narrative, following the example case study linked below.
- [x] Please read carefully checking for typos or grammatical errors (e.g. following sentence “The anonymization was achieved was replacing Twitter and voter unique IDs by randomly generated numeric IDs.”)
- [x]  Please define the acronym STAN

Diagram
- [x] Please make the _minimum_ font size in your diagram to be 10pt, and more ideally 11pt (space allowing), using Helvetica font (or similar).
- [x] Please add the labels “Stage I: Data Input”, “Stage II: Data Processing” and “Stage III: Data Analysis” to indicate the appropriate sections of your Diagram, following the examples in [CaseStudies123.png](https://github.com/BIDS/repro-case-studies/blob/submissions/CaseStudies123.png) as a guideline.  These labels should be in Helvetica font, bold, 22pt.
- [x] Please change any color/grayscale in your diagram to black & white.  There's just a small bit of grey, in the outlining shapes.  Can you change this to black?
- [x] Please provide both the PDF and editable source file for your Diagram.

Here is a traditional examples of a case study [Narrative](https://github.com/BIDS/repro-case-studies/blob/submissions/case-studies/dturek.md) and [Diagram](https://github.com/BIDS/repro-case-studies/blob/submissions/case-studies/dturek.pdf), which represent both the style and degree of "concreteness" that we're looking for.

**Comment by pablobarbera** [comment]
Thanks for the feedback! Pull request #102  introduces the following changes to address these comments:

Narrative
- The case study now has a title ("The trade-off between reproducibility and privacy in the use of social media data to study political behavior")
- I revised the case study and fixed all typos I found
- I added a URL link to the Stan reference. (Stan is not an acronym, or at least not as far as I know!)

Diagram
- The minimum size in the diagram in now 15pt with Helvetica.
- I added the three labels to indicate the different sections of the diagram, all in Helvetica 22pt bold.
- All the lines and shapes in the diagram are in black.
- I have added the diagram in PDF and SVG format as "barbera_edited.pdf" and "barbera_edited.svg".

Thanks again!

PS. Sorry, I think I accidentally added some old commits in the pull request. The changes to my files are only in the last commit within this pull request.
